.env.sample

```diff
@@ -1 +0,0 @@
-SALT=VERUCA
```

Comment:
Was the .env.sample conflicting with something? I should add `cp .env.sample .env` to the README (and also update the sample file), but we probably want to keep this one around.